labwork5/Args processing.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--year', help='Top250 movies titles with year',
```

Comment:
if you define arguments in this way I should print `python main.py --year whatever` to print titles with year. You can use `action="store_true"` option instead. 
It also concerns --rate and --all keys.